WebRoot/js/zimbraAdmin/accounts/controller/ZaAccountViewController.js

```diff
@@ -705,3 +709,16 @@ function(elementValue) {
 	}
 	return false;	
 }
+
+ZaAccountViewController.prototype._getLDAPAttr = email => {
```

Comment:
not sure if we should use arrow function, for older browsers this could be problematic

Reply:
changed it